microsoft-365/lighthouse/m365-lighthouse-overview-deployment-task.md

```diff
@@ -27,10 +27,10 @@ A tenant's deployment plan is complete when all tasks are compliant or explicitl
 
 |Factor|Description|
 |---|---|
-|Is the tenant licensed for the required services associated with the task?|<ul><li>Tasks that aren't licensed won't be eligible for deployment.</li><li>Tasks that aren't licensed may be dismissed.</li><li>Lighthouse will direct users to the Business Workshop Tool to determine licensing needs.</li><li>Once Lighthouse detects that the tenant is licensed for the required services associated with the task, the status will be updated automatically.</li></ul>|
-|Is the task related to other tasks?|<ul><li>Tasks related to other tasks must be completed in sequential order.</li><li>Subsequent tasks will be eligible for deployment once the pre-requisite tasks are compliant.</li></ul>|
+|Is the tenant licensed for the required services associated with the task?|<ul><li>Tasks that are not licensed won't be eligible for deployment.</li><li>Tasks that aren't licensed may be Dismissed.</li><li>Lighthouse will direct users to the Business Workshop Tool to determine licensing needs.</li><li>Once Lighthouse detects that the tenant is licensed for the required services associated with the task, the status will be updated automatically.</li></ul>|
```

Comment:
@pabloang1 - Is there a reason why you removed the contraction from "aren't" in the first bulleted item but left the contraction in "aren't" in the second one? Microsoft voice is to use contractions, so they should both be contractions. At the very least, they should be the same for consistency.

Reply:
Kyle's edits rolled back some contractions used. I'm now questioning if I should put them back.